docs/sources/shared/configuration.md

```diff
@@ -1209,19 +1209,20 @@ planner:
   # CLI flag: -bloom-build.planner.interval
   [planning_interval: <duration> | default = 8h]
 
-  # Newest day-table offset (from today, inclusive) to build blooms for.
-  # Increase to lower cost by not re-writing data to object storage too
-  # frequently since recent data changes more often at the cost of not having
-  # blooms available as quickly.
+  # Newest day-table offset (from today, inclusive) to build blooms for. 0 start
+  # building from today, 1 from yesterday and so on. Increase to lower cost by
+  # not re-writing data to object storage too frequently since recent data
+  # changes more often at the cost of not having blooms available as quickly.
   # CLI flag: -bloom-build.planner.min-table-offset
-  [min_table_offset: <int> | default = 1]
+  [min_table_offset: <int> | default = 0]
 
-  # Oldest day-table offset (from today, inclusive) to compact. This can be used
-  # to lower cost by not trying to compact older data which doesn't change. This
-  # can be optimized by aligning it with the maximum
-  # `reject_old_samples_max_age` setting of any tenant.
+  # Oldest day-table offset (from today, inclusive) to compact. 1 till
+  # yesterday, 2 till day before yesterday and so on. This can be used to lower
+  # cost by not trying to compact older data which doesn't change. This can be
+  # optimized by aligning it with the maximum `reject_old_samples_max_age`
+  # setting of any tenant.
   # CLI flag: -bloom-build.planner.max-table-offset
-  [max_table_offset: <int> | default = 2]
+  [max_table_offset: <int> | default = 1]
```

Comment:
Should we increase the default to 7? I think this would make sense in the broader context

Reply:
I'm hesitant to do that until we get more confident about how good the build process scales. If data doesn't change as often, building today + tomorrow should work more most of the cases.

Reply:
Ok with that